### PR TITLE
Improve the scancodeio index and getting started documentation #197

### PR DESCRIPTION
**Pull request by hyounes4560**
This PR: 

* Restructure the current index and getting started documentation

Signed-off-by: Hanan Younes <hyounes4560@conestogac.on.ca>

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ScanCode.io documentation
 =========================
 
-ScanCode.io is a server to script and automate software composition analysis
-with ScanPipe pipelines.
-First application is for Docker container and VM composition analysis.
+Welcome to the very start of your ScanCode.io journey! In this documentation you’ll find information on:
+
+- An overview of ScaCode.io and ScanPipe
```
Comment (AyanSinhaMahapatra) [inline docs/index.rst:6]:
```suggestion
- An overview of ScanCode.io and ScanPipe
```

File: docs/introduction.rst [hunk 2]
```diff
@@ -1,77 +1,73 @@
...
-And when you reuse only a few FOSS components in a single project, running one
-of these tools (such as the scancode-toolkit) by hand together
-with a spreadsheet may be enough to manage your software composition analysis.
+- You're required to know the license of third-party code before using it, **and**
+- You want to avoid using buggy, outdated or vulnerable components.
 
-But when you scale up, running automated and reproducible analysis pipelines
-that are adapted to a software project unique context and technology platform is
+It's usually convenient to include and reuse new code downloaded from the internet; however, it's often surprisingly hard to get a proper inventory of all third-party code origins and licenses used in a software project.
+There are some great tools available to scan your code and help uncover these details. For example, when you reuse only a few FOSS components in a single project, running one of these tools, such as the **ScanCode-toolkit**, manually along
```
Comment (AyanSinhaMahapatra) [inline docs/introduction.rst:21]: It would make sense to have line break at 100 characters, this also fails the tests as the linter chacks for those.

Comment (AyanSinhaMahapatra) [reply docs/introduction.rst:21]: Running `make check` locally before pushing would be a good thing to do.

**Review by AyanSinhaMahapatra — COMMENTED** [review]
You can stage new changes and amend your previous commit to avoid having to make several commits.

**Comment by tdruez** [comment]
> @AyanSinhaMahapatra You can stage new changes and amend your previous commit to avoid having to make several commits.

I would not recommend that approach. Please commit/push as often as you need to "save" your progress, as long as you are in a branch, since we will Squash all those commits into a single one before the merge into the main branch.

**Review by tdruez — APPROVED** [review]

**Pull request merged**